src/sbt-test/sbt-header/simple/build.sbt

```diff
@@ -0,0 +1,21 @@
+import de.heikoseeberger.sbtheader.license.Apache2_0
```

Comment:
Strangely our comments are no longer put underneath the respective code, hence it's hard for me to understand your questions.

What I was trying to say is that you should only use the `:=` operator, not `<<=` and friends.